packages/syntax-core/src/Card/Card.tsx

```diff
@@ -28,7 +28,7 @@ type CardType = {
  */
 export default function Card({
   children,
-  size = "sm",
+  size,
```

Comment:
do you want to get rid of the size prop entirely and have devs just manually input maxWidth?

Reply:
Hmmm I'm not that opposed to it, but this `size` interface still feels simpler to me